lab/salvagehost-dev/validate-usb-reconciler.sh

```
@@ -0,0 +1,199 @@
+fi
+grep -q 'qemu group does not exist' "$tmp_dir/missing-group.out" || fail "missing group refusal was not explicit"
+
+if grep -Eq 'systemctl (enable|start) salvagehost-usb-reconcile\.timer' \
+  "$script_dir/install-usb-reconciler.sh"; then
+  fail "installer still enables or starts the periodic timer"
+fi
+if grep -Eq 'install .*rendered_udev.*install_udev|install -m [0-9]+ .*rendered_udev.*udev/rules\.d' \
+  "$script_dir/install-usb-reconciler.sh"; then
+  fail "installer still installs a persistent udev rule"
+fi
+grep -q 'install -m 0755 "\$monitor" "\$install_monitor"' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)